ci-operator/config/openshift-service-mesh/sail-operator/openshift-service-mesh-sail-operator-main.yaml

```
@@ -162,15 +163,15 @@ tests:
           memory: 200Mi
       timeout: 2h0m0s
     workflow: servicemesh-istio-e2e-hypershift
-- as: e2e-ocp-4-17
+- as: e2e-ocp-4-18
   optional: true
   steps:
     cluster_profile: ossm-aws
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:next
```

```suggestion
      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
```

This tag is from openshift-cluster-control-plane-machine-set-operator-release-4.18_periodics.yaml
It should be a 4.18 version tag.